miniml/compiler/compile.scm

```diff
@@ -256,6 +257,7 @@
     (EQ) : "="
     (MINUS) : "-"
     (STAR) : "*"
+    (PERCENT) : "%"
     (INFIXOP0) : $1
```

Comment:
Could you also add a case `expr_no_semi PERCENT expr_no_semi` below and set the priority and associativity of 'PERCENT` to be the same as `STAR`? Right now `e1 % e2` is no longer a valid expression.

Reply:
If it's not in the testsuite, it does not exist :-)

Reply:
Done.